docs/ota-client-guide/modules/ROOT/pages/posix-secondaries-bitbaking.adoc

```diff
@@ -4,7 +4,7 @@ The goal of this doc is to guide a reader on bitbaking of two type of images `pr
 and running of which on the target makes it act as `Primary` and `Secondary` ECU of a single device.
 
 Please, refer to link:https://uptane.github.io/uptane-standard/uptane-standard.html[the Uptane standard] in order to grok the meaning of the `Primary` and `Secondary` terms in the given context from a theoretical standpoint.
-Please, check out link:posix-secondaries.doc[this doc] to understand these terms from a practical standpoint and
+Please, check out xref:posix-secondaries.doc[this doc] to understand these terms from a practical standpoint and
```

Comment:
Commas after Please here are a bit weird.

Reply:
that weren't me! but fixing now.